### DISCUSSION TITLE
Why is my espresense-companion Calibration screen empty?

### DISCUSSION THREAD

**Discussion by RaoulSargent** [ESPresense/ESPresense-companion#446] [Q&A] [answered]
I am looking for help as to why my Calibration screen contains no information, totally empty.
What am I missing?

Other than the calibration screen, the map/devices/nodes screens all seem to be operating as I would expect.

Running Companion v1.5.1 (downloaded container from Docker Hub on Sun 3rd Dec 2023).

I have:
1x esp32 D1 mini running espresense v3.3.2b3
14x esp32 D1 mini running espresense v3.3.1
1x esp32 D1 mini running espresense v3.2.4

My Calibration screen looks like this:
<img width="1410" alt="image" src="https://github.com/ESPresense/ESPresense-companion/assets/61843699/e5ad3687-20e1-407c-a2ba-d0a93cb4a352">

And here is a snippet from my configuration file:
<img width="445" alt="image" src="https://github.com/ESPresense/ESPresense-companion/assets/61843699/dcb2b7f8-e0a8-49ce-9003-f311a0547154">

**Comment by DTTerastar** [comment]
Did you put some stuff in include on the nodes but not put in node: ?  Those are what drive that screen.

**Comment by RaoulSargent** [comment]
To limit the mqtt messaging, on each esp32 node I have put the same list of the iphones/watches I wish to track in the "Filtering / Include" field.
example:  "R_iPhone R_watch V_iPhone V_watch"

I also have the following in the "Filtering / Exclude" field:
"exp:20 sd: msft sonos apple samsung name:"

I didn't quite understand the second part of your answer, "_...but not in the node_"
Are you saying I need to add the "?" (question mark) in the include field?

**Comment by RaoulSargent** [comment, marked as answer]
Further to my previous reply I have now added each esp32 'name' to the include field list on all nodes,

and the following line in the config file:

- id: “node:*”

Now starting to see info in the calibration page.
